src/content/reference/cluster-upgrades/index.md

```diff
@@ -0,0 +1,190 @@
+This ensures that the stack running in your environment is the exact stack we have tested before.
+
+As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new release.
+
+#### Release upgrade semantics
+
+As the semantic versioning system defines, an upgrade from one version to another is either one of these:
+
+- *Patch*: The smallest type of upgrade occurs when we publish bugfixes, security fixes, or make changes to the observability while maintaining the given functionality of the stack.
+This can include any sort of patch upgrades of third party components.
+
+- *Minor*: A minor upgrade occurs when we add functionality, while maintaining the functionality of the stack as it was in the previous version.
+This can include minor upgrades of third party components, with the exception of Kubernetes.
+
+- *Major*: A major upgrade occurs when (A) a minor Kubernetes version upgrade is picked up (e. g. from 1.10.* to 1.11.*), when we remove functionality or change functionality that might require changes in workloads, in automation working with the cluster or in administrator's interactions.
+
+**Note:** The release version number is provider-specific.
+Azure, AWS, and KVM based installations have independent versioning systems, as their stacks are also slightly different.
+
+### Determining the release version and inspecting release details
+
+As a user with access to the Giant Swarm API and being member of the organization owning the cluster, you can use both the web UI or the CLI to find out what release version your tenant cluster has currently.
+
+In the web UI, both the cluster overview and the cluster details page show the release version number. 
+In the cluster details page you can click the release version number to get more information about a release.
+
+Likewise in the CLI, commands like [`gsctl list clusters`](/reference/gsctl/list-clusters/) and [`gsctl show cluster`](/reference/gsctl/show-cluster/) reveal the release version number.
+To get information on all available releases, use the [`gsctl list releases`](/reference/gsctl/list-releases/) command.
+
+### Release lifecycle
+
+**We hold two different major releases** available at any time to chose from when creating a new cluster, which means that you have two different Kubernetes minor versions to chose from.
+
+Whenever a new Kubernetes minor version is released by the Kubernetes project, we aim to make that version available in a new major release of our stack within 30 days from the Kubernetes release.
+
+Once we publish a new major release, we deprecate the oldest major release.
+This means that no new clusters can be created using that old release version.
+Existing clusters, however, are not affected.
+
+**Both patch and minor upgrades** can be rolled out at any time by Giant Swarm without your interaction. Currently, this happens in coordination with your administrators and with a notice to your developers.
```

Comment:
`administrators` -> `Solution Engineers`?

Reply:
no the SEs communicate with the admins of the customer